pkgs/by-name/ad/adminer-pematon/index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace nixos {
```

Comment:
Opinionated: I would modify this into something more functional, like:

```php
<?php

namespace nixos {
	function adminer_object(): object {
		require_once(__DIR__ . '/plugins/plugin.php');

		if (!file_exists(__DIR__ . '/plugins.json')) {
		    return new \AdminerPlugin();
		}

        $plugins = array_map(
            static function (string $name): ?object {
    			$plugin = sprintf('%s/plugins/%s.php', __DIR__, $name);

    			if (!is_readable($plugin)) {
    			    return null;
    			}

				require($plugin);

				preg_match_all('/(\w+)/', $name, $matches);

				return new sprintf('Adminer%s', implode('', array_map('ucfirst', $matches[1])));
            },
            json_decode(file_get_contents(sprintf('%s/plugins.json', __DIR__), true)
        );

		return new \AdminerPlugin(array_filter($plugins));
	}
}
```

Reply:
Done, thanks, tested and it works.

One thing I might add though, do you know perhaps a way to use [adminer-theme](https://github.com/pematon/adminer-theme) here?

I am currently [removing the index.php](https://github.com/JohnRTitor/nix-conf/blob/57e2dd255de05a6f538d27352c9110693a83372e/pkgs/adminer-pematon-with-adminer-theme/default.nix#L26-L27) provided by this package, and adding an index.php with [modifications for adminer theme](https://github.com/JohnRTitor/nix-conf/blob/57e2dd255de05a6f538d27352c9110693a83372e/pkgs/adminer-pematon-with-adminer-theme/index.php#L25-L32).